libraries/mbed/targets/hal/TARGET_WIZNET/TARGET_W7500x/us_ticker.c

```diff
@@ -68,66 +86,91 @@ void us_ticker_init(void)
     us_ticker_inited = 1;
 
     SystemCoreClockUpdate();
-    TimMasterHandle_CH3.PWM_CHn_PR = (GetSystemClock() / 1000000) -1;
-    TimMasterHandle_CH3.PWM_CHn_LR = 0xFFFFFFFF;
-    TimMasterHandle_CH3.PWM_CHn_PDMR = 1;
+    TimerInitType.PWM_CHn_PR = (GetSystemClock() / 1000000) -1;
+    TimerInitType.PWM_CHn_LR = 0xFFFFFFFF;
+    TimerInitType.PWM_CHn_PDMR = 1;
 
-    PWM_TimerModeInit(PWM_CH3, &TimMasterHandle_CH3);
-    PWM_CHn_Start(PWM_CH3);
+    PWM_TimerModeInit(TIMER_1, &TimerInitType);
+    PWM_CHn_Start(TIMER_1);
 }
 
 
 uint32_t us_ticker_read()
 {
     if (!us_ticker_inited) us_ticker_init();
-
-    return (PWM_CH3->TCR);
+    return (TIMER_1->TCR);
 }
 
 
 void us_ticker_set_interrupt(timestamp_t timestamp)
 {
     int32_t dev = 0;
+    
     if (!us_ticker_inited)
     {
         us_ticker_init();
     }
     
-    dev = (int32_t)(timestamp - (us_ticker_read() + 150));
-
+    dev = (int32_t)(timestamp - (us_ticker_read() + 160));
```

Comment:
1. I use two timer in us_ticker.c, one is PWM timer and one is dualtimer.
PWM timer has integer prescaler but dualtimer has prescaler 1, 16 and 256.
And I use 16 prescaler and default core clock is 20MHz. PWM timer use prescaler 20 so it counts 1 every 1us. 
But Dualtimer is not. It counts 1 every 0.8us so I adjust time as 1 us by multiplying 1.27.